backend/app/libs/storage_helpers/treasury_cacher.py

```diff
@@ -0,0 +1,45 @@
+from json import JSONDecodeError
```

Comment:
I guess this shouldn't go with `storage_helpers.py` which was just a technical module. But anyways I'm suggesting a modules structure change in my next PR and we'll start factoring things from there bit by bit. Let's see which PR gets merged first.